src/engine/NodeHelpers.cs

```diff
@@ -69,6 +69,24 @@ public static void QueueFreeChildren(this Node node, bool detach = true)
         }
     }
 
+    /// <summary>
+    ///   Call RemoveChild on all Node children
+    /// </summary>
+    /// <param name="node">Node to remove children of</param>
+    public static void RemoveChildren(this Node node)
+    {
+        while (true)
+        {
+            int count = node.GetChildCount();
+
+            if (count < 1)
+                break;
+
+            var child = node.GetChild(count - 1);
+            node.RemoveChild(child);
+        }
+    }
+
```

Comment:
Wouldn't a decreasing `for` loop be nicer and more efficient than this while loop?

Reply:
Yeah, but I also just forgot to remove this function